### DISCUSSION TITLE
How to turn off auto-completion of backquotes to backquotes + single quotes？

### DISCUSSION THREAD

**Discussion by yaohong772** [James-Yu/LaTeX-Workshop#3108] [Q&A]

**Comment by tamuratak** [comment]
You cannot. VS Code doesn't allow it configurable. See microsoft/vscode/issues/38352

**Comment by tamuratak** [comment]
Please fill out the [issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/feature-request.md) next time.

**Comment by jlelong** [comment]
There is now a separate setting for closing quotes. I think it is sufficient to set `editor.autoClosingQuotes` to `never` for the `latex` language and backticks, which are considered as quotes, are no more auto closed.
